src/pages/AppBrowser/AppBrowser.js

```diff
@@ -3,7 +3,7 @@ export default {
   data () {
     return {
       loading: true,
-      path: `http://${this.$route.fullPath}`
+      path: `https:/${this.$route.fullPath}`
```

Comment:
so all aepps and pages have to be https? is that intended?

Reply:
Yes.

> For security reasons, web browser disallows opening of websites in an iframe by http when the whole page loaded by https. (7a22e6a0150cbdc8724374f111fc75a4455c6204 commit message)

https is necessary for the Base app for WerRTC and Web crypto in Chrome browser.
But it can be very uncomfortable in development, for example when all is on localhost and uses http, I have added a fix.